its/ruling/src/test/resources/expected/python-S1607.json

```diff
@@ -0,0 +1,26 @@
+{
+  'project:mypy-0.782/mypy/test/data.py':[
+    217,
+  ],
+  'project:mypy-0.782/mypy/test/testfinegrained.py':[
+    77,
+  ],
+  'project:mypy-0.782/mypy/test/testipc.py':[
+    59,
```

Comment:
This case (as well as a few others from ruling) puzzles me a bit.
The developer is not providing a reason in the arguments for the call, but there is still a reason provided through comments above the decorator.
We may want to avoid reporting issues for this case (to be verified, as an explicit skip through pytest/unittest call might provide some additional info when running all the test suite).

Personally, I think it's fine if we report an issue in such case, but I'd probably add a test case in our UTs to document the behavior.

Reply:
That's a good point, I added the test case with some comment.
I personally agree with you, for me we should indeed raise an issue as I believe the reason should really be in the method/decorator rather than in the comment, but probably both points have valid arguments.